src/NUnitFramework/framework/Internal/Reflect.cs

```diff
@@ -24,7 +24,11 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.ComponentModel;
```

Comment:
Major nit pick :). Is this necessary?

Reply:
Nice spot! I just had a thought– maybe we should leave some time for the contributor to respond if there are questions before merging? :D

Reply:
Actually, I did not find this important enough to hold back the PR, and perhaps I should just have ignored it :)